adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -1121,6 +1072,8 @@ private class NoPrefixFileFilter(private val prefix: String) extends PathFilter
  * @param sc The SparkContext to wrap.
  */
 class ADAMContext(@transient val sc: SparkContext) extends Serializable with Logging {
+  @transient val spark = SQLContext.getOrCreate(sc).sparkSession
```

Comment:
This doesn't appear to be used anywhere, except for the import below.  Why bring that up here instead of keeping it as `import sqlContext.implicits._` within a method as before?

Reply:
Well, I thought it was a reasonable thing to have available, and it saved me from importing the implicitits in each of the new functions I added. I'm fine with moving it if you prefer, though.

Reply:
If it eliminates the need to repeatedly import the implicits, then I'd favor keeping it.